src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -163,6 +166,36 @@ const FileInUseModal = ({ onDismiss, onCopy, onPlayground, namespace, name, buck
   ])
 }
 
+const EditModeDisabledModal = ({ onDismiss, onRecreateCluster, onPlayground }) => {
```

Comment:
TOL: This page has become very modal-heavy, which is a different pattern than how we handle these kinds of messages elsewhere on the site. We should think this over going forward.